build/Targets/Settings.targets

```diff
@@ -97,6 +102,10 @@
     <UseRoslynAnalyzers Condition="'$(UseRoslynAnalyzers)' == ''">true</UseRoslynAnalyzers>
 
     <Deterministic>True</Deterministic>
+
+    <!-- Only portable PDBs are supported xplat -->
+    <DebugType Condition="'$(OS)' != 'Windows_NT'">portable</DebugType>
+    <DebugType Condition="'$(OS)' == 'Windows_NT'">pdbonly</DebugType>
```

Comment:
It's all happening!!!  

CC @tmat our Linux / Mac jobs now going to produce portable PDB on every run.